src/plugins/data/public/utils/table_inspector_view/components/data_view.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+import { DataTableFormat } from './data_table';
+import { IUiSettingsClient } from '../../../../../../core/public';
+import {
+  Datatable,
+  DatatableColumn,
+} from '../../../../../expressions/common/expression_types/specs';
+import { InspectorViewProps, Adapters } from '../../../../../inspector/public';
+import { UiActionsStart } from '../../../../../ui_actions/public';
```

Comment:
nit - since these are all types you can also save yourself the headache and use absolute paths: `src/core/public`, `src/plugins/inspector/public` etc